application/models/TemplateConfig.php

```diff
@@ -300,9 +300,9 @@ public function throwConsoleError($sCustomMessage = null)
 
 
     /**
-     * @todo document me
-     * @todo missing return value (php warning)
-     * @return boolean|null
+     * Check if this template is a standard template and save it in current model $this->isStandard
```

Comment:
Using "and" in a function description is a tell that it should rather be two functions. But it's old, right?